README.md

```diff
@@ -5,61 +5,68 @@ GPU infrastructure and automation tools
 
 ## Overview
 
-The DeepOps project facilitates deployment of GPU servers and multi-node GPU clusters for Deep Learning and HPC environments, in an on-prem, optionally air-gapped datacenter or in the cloud.
+The DeepOps project encapsulates best practices in the deployment of GPU server clusters and sharing single powerful nodes (such as [NVIDIA DGX Systems](https://www.nvidia.com/en-us/data-center/dgx-systems/)). DeepOps can also be adapted or used in a modular fashion to match site-specific cluster needs. For example:
 
-Use the provided Ansible playbooks and scripts to deploy Kubernetes, Slurm, or a hybrid of both. This repository encapsulates best practices to make your life easier, but can also be adapted or used in a modular fashion to suite your specific cluster needs. For example: if your organization already has Kubernetes deployed to a cluster, you can still use the optional services and scripts provided to install Kubeflow, enable authentication, or connect NFS storage.
+* An on-prem, air-gapped data center of NVIDIA DGX servers where DeepOps provides end-to-end capabilities to set up the entire cluster management stack
+* An existing cluster running Kubernetes where DeepOps scripts are used to deploy Kubeflow and connect NFS storage
+* An existing cluster that needs a resource manager / batch scheduler, where DeepOps is used to install Slurm, Kubernetes, or a hybrid of both
 
-> NOTE: we recommend using the most recent release branch for stable code.
-> The `master` branch is used for development and as such may be unstable or even broken at any point in time.
+## Releases
+
+Latest release: [DeepOps 19.03 Release](releases/tag/19.03)
+
+It is recommended to use the latest release branch for stable code (linked above). The master branch is generally stable, but is used for development.
```

Comment:
> The master branch is generally stable, but is used for development.

Master is not really "stable", in the sense that someone who frequently updates from master might find a lot changing under them. :) Maybe re-phrase to something like,

> All development takes place on the master branch, which is generally functional but may change significantly between releases.